vuepress/docs/v7.0/tutorials/compose/content-templates-tutorial.md

```diff
@@ -1,32 +1,32 @@
 # Content Templates
 
-Content Templates define how information is displayed when a Content is published. They provide the styling and layout for Content Types and offer different representations of the same content.
+Content templates define how information is displayed when content is published. They provide the styling and layout for content types and enable different representations of the same content.
```

Comment:
Hate to say it, but I think it should be Content Template and Content Type in this context.  They are features in Entando and capitalized in the UX. :(

Reply:
no worries at all... the capitalization struggle with these specific terms has been real :(